app/goal_routes.py

```diff
@@ -0,0 +1,94 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from app.task_routes import validate_model
+from datetime import datetime
+import requests
+import os
+
+goals_bp = Blueprint("goals_bp",__name__, url_prefix="/goals")
+
+@goals_bp.route("", methods=["POST"])
+def post_goal():
+    request_body = request.get_json()
+    
+    if not request_body.get("title"):
+        return jsonify({"details": "Invalid data"}), 400
+    
+    new_goal = Goal(
+        title=request_body["title"]
+        )
+
+    db.session.add(new_goal)
```

Comment:
Great job using `db.session.<method-name>`. Session gives us access to the follow:

- `db` is our app’s instance of SQLAlchemy.
- `session` represents our active database connection.
- By referencing `db.session` we can use SQLAlchemy’s methods to perform tasks like:
    - committing a change to a model
    - storing a new record of a model
    - deleting a record.
- By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the task model